Mathlib/Algebra/Category/BialgebraCat/Monoidal.lean

```diff
@@ -0,0 +1,71 @@
+/-
+Copyright (c) 2024 Amelia Livingston. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Amelia Livingston
+-/
+
+import Mathlib.Algebra.Category.AlgebraCat.Monoidal
+import Mathlib.Algebra.Category.BialgebraCat.Basic
+import Mathlib.Algebra.Category.CoalgebraCat.Monoidal
+import Mathlib.RingTheory.Bialgebra.TensorProduct
+
+/-!
+# The monoidal structure on the category of bialgebras
+
+In `Mathlib.RingTheory.Bialgebra.TensorProduct`, given two `R`-bialgebras `A, B`, we define a
+bialgebra instance on `A ⊗[R] B` as well as the tensor product of two `BialgHom`s as a
+`BialgHom`, and the associator and left/right unitors for bialgebras as `BialgEquiv`s.
+In this file, we declare a `MonoidalCategory` instance on the category of bialgebras, with data
+fields given by the definitions in `Mathlib.RingTheory.Bialgebra.TensorProduct`, and Prop
+fields proved by pulling back the `MonoidalCategory` instance on the category of algebras,
+using `Monoidal.induced`.
+
+-/
+
+universe v u
```

Comment:
It seems you need only one universe:
```suggestion
universe u
```

Reply:
Thanks for the comments! I have applied them.